app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -333,23 +332,43 @@ export default class CoverageVizBottomSidebar extends React.Component {
       referenceLength: currentAccessionData.total_length,
       alignedContigs: currentAccessionSummary.num_contigs,
       maxAlignedLength: currentAccessionData.max_aligned_length,
-      coverageDepth: currentAccessionData.coverage_depth,
-      coverageBreadth: currentAccessionData.coverage_breadth,
+      coverageDepth: `${currentAccessionData.coverage_depth}x`,
+      coverageBreadth: `${100 * currentAccessionData.coverage_breadth}%`,
```

Comment:
nit: Should percent be a function in case we need to enforce changes (e.g. precision) for all?